dist/docs.json

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
Why do you want to check in the dist folder?

Reply:
Because otherwise, you cannot install the plugin from git.

Reply:
ok, but I don't want to support installing from git. People should use npm for that. Can you please delete the folder and add it again to .gitignore?

Reply:
npm install doesn't work without support installing from git. This is why the dist (distribution) folder for.